backend/src/poap/index.js

```diff
@@ -1,15 +1,35 @@
 import axios from 'axios';
 
 class POAP {
+    poapApiHttpClient;
 
+    /**
+     * @param apiKey The POAP API API-Key: see {@link https://documentation.poap.tech} for more information
+     */
+    constructor(apiKey) {
+        this.poapApiHttpClient = axios.create({
+            baseURL: 'https://api.poap.tech',
```

Comment:
I think you prefered to leave like this right? without using an env var for this url